src/assets/svgs.js

```diff
@@ -1,31 +1,7 @@
-import React from "react"
+import FaceBookIcon from './FacebookIcon';
+import TwitterIcon from './TwitterIcon';
```

Comment:
remove semi-colons from all changed files because we are using the no-semi-colon approach